Template/NLO/SubProcesses/setcuts.f

```diff
@@ -416,7 +416,11 @@ subroutine set_tau_min()
                endif
                xw(i)=0d0
             enddo
-            stot = 4d0*ebeam(1)*ebeam(2)
+            if (ebeam(1).le.0.938.or.ebeam(2).le.0.938) then 
+                stot=2d0*ebeam(1)*ebeam(2)
+            else 
+                stot=4d0*ebeam(1)*ebeam(2)
+            endif
```

Comment:
This need at least a comment to explain "why", you change the formula...
Should not we use the correct formula (without any assumptions)?